ui/src/common/layouts/Header/Header.jsx

```diff
@@ -10,53 +10,50 @@ import BetaInfoBanner from './BetaInfoBanner';
 import InterventionBanner from './InterventionBanner';
 import HeaderMenuContainer from './HeaderMenuContainer';
 import BetaRibbon from './BetaRibbon';
+import CollectionsMenu from '../CollectionsMenu/CollectionsMenu';
 
-function Header(props) {
-  const { onHeightChange } = props;
-  const [ref, , height] = useResizeObserver();
+function Header({ isHomePage, isSubmissionsPage, isBetaPage }) {
+  const [stickyContainerRef, , stickyContainerHeight] = useResizeObserver();
 
-  useEffect(
-    () => {
-      onHeightChange(height);
-    },
-    [height, onHeightChange]
-  );
-
-  const { isHomePage, isSubmissionsPage, isBetaPage } = props;
   return (
-    <div ref={ref} className="__Header__">
-      <InterventionBanner />
-      {isBetaPage && (
-        <>
-          <BetaInfoBanner />
-          <BetaRibbon />
-        </>
-      )}
-      <Layout.Header className="header">
-        <Row type="flex" align="middle" gutter={{ xs: 8, sm: 16 }}>
-          <Col xs={{ span: 12, order: 1 }} sm={{ span: 6, order: 1 }} lg={5}>
-            <Logo />
-          </Col>
-          <Col
-            xs={{ span: 24, order: 3 }}
-            sm={{ span: 14, order: 2 }}
-            lg={12}
-            xl={13}
-            xxl={14}
-          >
-            {!isHomePage && !isSubmissionsPage && <SearchBoxContainer />}
-          </Col>
-          <Col
-            xs={{ span: 12, order: 2 }}
-            sm={{ span: 4, order: 3 }}
-            lg={7}
-            xl={6}
-            xxl={5}
-          >
-            <HeaderMenuContainer />
-          </Col>
-        </Row>
-      </Layout.Header>
+    <div className="__Header__">
+      <div ref={stickyContainerRef} className="sticky">
+        <InterventionBanner />
+        {isBetaPage && (
+          <>
+            <BetaInfoBanner />
+            <BetaRibbon />
```

Comment:
Can we just get rid of them from now?

Reply:
https://its.cern.ch/jira/browse/INSPIR-3288

Reply:
I think it's better to do it separately, it's not just removing the ribbon, but also modifying the banner from `beta` to `new release`, with different info in it.